mangum/adapter.py

```diff
@@ -93,9 +95,20 @@ def __post_init__(self, enable_lifespan: bool) -> None:
 
         self.logger: logging.Logger = get_logger(self.log_level)
 
+    @staticmethod
```

Comment:
The behaviour of the `get_header` method in the HTTP protocol class is similar enough that this should be moved out into a new `utils.py` file and then imported in both files. The method is otherwise fine. :)